src/common/test-runtime/project.json

```diff
@@ -9,7 +9,7 @@
     "System.Runtime.InteropServices.RuntimeInformation": "4.4.0-beta-24813-03",
     "System.Linq.Parallel": "4.0.1",
     "OpenCover": "4.6.519",
-    "ReportGenerator": "2.5.0",
+    "ReportGenerator": "2.5.9",
```

Comment:
While it is probably find to update this version we don't really use this in corefx any longer. Once you integrate this change into corefx you will need to update. https://github.com/dotnet/corefx/blob/master/external/test-runtime/XUnit.Runtime.depproj#L62, ideally you should change it to use the msbuild property $(ReportGeneratorVersion) now.